### ISSUE TITLE
Combo-box: Remove aria-describedby reference to deprecated USWDS element

### ISSUE THREAD

**Issue by amyleadem** [department-of-veterans-affairs/vets-design-system-documentation#6053]
## Description
**Remove the reference to `assistiveHintID` in the `va-combo-box component`.** The reference is now invalid since it points at a now-deprecated element from the USWDS combo box component that no longer exists.

[Related Slack thread](https://dsva.slack.com/archives/C0335Q88X0S/p1775071964663319) (🔒)

### Details
In USWDS v3.9.0, [they removed the assistive hint text that was in an sr-only span from its combo box component](https://github.com/uswds/uswds/pull/6022). To match the USWDS component behavior in the VADS web component, the VADS combo-box had to hard-code a reference to that generated element into the component's `aria-describedby` attribute. This reference should now be removed since USWDS no longer generates that span. 

## Tasks
- [ ] Remove the [`assistiveHintID` variable declaration](https://github.com/department-of-veterans-affairs/component-library/blob/dfa76f4fa372bc02ee92ce648083998231998068/packages/web-components/src/components/va-combo-box/va-combo-box-library.js#L189) and related code
- [ ] Confirm no regressions in the `aria-describedby` behavior and `messageAriaDescribedby` prop

## Acceptance Criteria
- [ ] The `va-combo-box` component no longer has any invalid values hard-coded into the `aria-describedby` attribute
